src/sentry/models/integration.py

```diff
@@ -145,6 +150,14 @@ def add_organization(self, organization, user=None, default_auth_id=None):
             if not created and default_auth_id:
                 org_integration.update(default_auth_id=default_auth_id)
         except IntegrityError:
+            logger.info(
```

Comment:
Up to you, but if you want, you could make this a `logger.error` and then it'll show up in Sentry.

Reply:
@manuzope Since this is a caught error, I don't think it makes sense to add a `logger.error` here.